zc_events/client.py

```diff
@@ -498,6 +502,22 @@ def send_email(self, *args, **kwargs):
 
         self.emit_microservice_email_notification('send_email', **event_data)
 
+    def send_slack_notification(self, *args, **kwargs):
+        canonical_user_id = kwargs.get('canonical_user_id')
+        notification_type = kwargs.get('notification_type')
+        application = kwargs.get('application')
+        message_data = kwargs.get('data')
```

Comment:
I might get to this when I'm doing the other PR's, but are we not using title & body at all for slack notifications? Wondering if we might want to log either of those if we are. Obviously no need if we're not.

Reply:
We are not using title & body.

The reason we use title and body in catering-core is b/c we are sending the whole message, formatted, to firebase (which does not know how to do any further processing).

For Slack, we are sending from catering-core to ultimately be processed by Slack App - which does have the logic to fetch the necessary data & also build/format the messages separately.